BDPointAndroidXamarinDemo/BroadcastReceiver.cs

```diff
@@ -12,17 +12,33 @@ public class AppGeoTriggerReceiver : GeoTriggeringEventReceiver
     {
         public override void OnZoneEntryEvent(ZoneEntryEvent entryEvent, Context context)
         {
-            Toast.MakeText(context, "Received OnZoneEntryEvent "+entryEvent.ZoneInfo.ZoneName, ToastLength.Short).Show();
+            ZoneInfo zoneInfo = entryEvent.ZoneInfo;
+            string zoneData = "";
+            foreach(KeyValuePair<string, string> entry in zoneInfo.CustomData)
+            {
+                zoneData += $"{entry.Key}: {entry.Value}\n";
+            };
```

Comment:
I would personally prefer a `map` function to explicitly return a value.

Reply:
Same, but the c# Dictionary doesn't appear to have a built in map function.